tests/powershell/Integration/AzureDev.Lifecycle.PublicCommand.TestHelper.ps1

```
@@ -0,0 +1,464 @@
+  param(
+    [Parameter(Mandatory = $true)]
+    [System.Management.Automation.PSObject]$Fixture
+  )
+
+  $recordPaths = @(Get-ChildItem `
+      -LiteralPath (Join-Path $Fixture.RepositoryRoot '.azure/logs') `
+      -Filter '*.jsonl' `
+      -File).FullName
+  return @($recordPaths | ForEach-Object {
+      Get-Content -LiteralPath $_ -Raw | ConvertFrom-Json
+    })
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-AzureDevLifecyclePublicCommandRecords' returns an object of type 'System.Object[]' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)